COMPILING

```diff
@@ -36,7 +36,7 @@ We assume that you have a Debian/Ubuntu or Red Hat-like distribution.
 
    On Red Hat/Fedora or derivates, do
 
-   yum install gcc gcc-c++ flex bison perl-libwww-perl patch
+   yum install gcc gcc-c++ flex bison perl-libwww-perl patch devtoolset-6
```

Comment:
Removing gcc and gcc-c++ makes the regression tests fail without the devtoolset-6 enabled. I cannot immediately see if this is a problem with the tests or with the cbmc binary. @kroening please let me know if you would like me to investigate further or you are happy with the instructions as is.